server/docker-compose.yml

```diff
@@ -13,6 +13,8 @@ services:
       - $POSTGRES_PORT:$POSTGRES_PORT
     networks:
       - app-network
+    volumes:
+      - ${PWD}/init_db.sql:/docker-entrypoint-initdb.d/init_db.sql
```

Comment:
Maybe use absolute path

Reply:
tried already, will give it another run ...

Reply:
This requires the docker compose file to be run inside the server directory